sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryAsyncClient.java

```diff
@@ -505,6 +506,88 @@ public DirectoryAsyncClient getSubDirectoryClient(String subDirectoryName) {
         return new PagedFlux<>(() -> retriever.apply(null), retriever);
     }
 
+    /**
+     * Closes a handle on a directory or a file at the service. This is intended to be used alongside
+     * {@link #listHandles(Integer, boolean) listHanldes}.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Force close handles with handles returned by get handles in recursive.</p>
+     *
+     * {@codesnippet com.azure.storage.file.directoryAsyncClient.forceCloseHandles}
+     *
+     * <p>For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles">Azure Docs</a>.</p>
+     *
+     * @param handleId Handle ID to be closed.
+     * @return An empty response.
+     */
+    public Mono<Void> forceCloseHandle(String handleId) {
```

Comment:
Wrap this with try-catch and return error in Mono.error(). Do the same for all other public methods that return Mono/Flux/PagedFlux.

Reply:
Done